reconcile/gql_definitions/oum/organizations.gql

```diff
@@ -0,0 +1,14 @@
+# qenerate: plugin=pydantic_v1
+
+query OUMOrganizations($name: String) {
```

Comment:
is `$name` used anywhere? There is one client side filter by env name, can we do server side filter?

Reply:
It is not needed. Remove in [d590cc8](https://github.com/app-sre/qontract-reconcile/pull/4771/commits/d590cc8291cf1f8b0ce3951bbd99e73d709909dd)